api/resource/resource_test.go

```diff
@@ -1400,30 +1400,30 @@ spec:
   numReplicas: 1
 `))
 	assert.NoError(t, err)
-	r.AppendRefBy(resid.FromString("gr1_ver1_knd1|ns1|name1"))
+	r.AppendRefBy(resid.FromString("gr1.ver1.knd1/ns1.name1"))
```

Comment:
nit: these placeholder values are in the wrong order... the test still shows they round-trip correctly, but it's confusing

Reply:
Fixed!